wurst/systems/spells/mage/elementalist/Brambles.wurst

```diff
@@ -196,8 +194,8 @@ class BramblesInstance
                         if not u.hasAbility(DUMMY_SLOW_BUFF)
                             new IttDummyCaster().castTarget(owner, DUMMY_SLOW_SPELL_ID, 1, Orders.slow, u, ownerUnit.getPos())
                         if not snaredTargets.has(u) and isOvercharged
-                            new IttDummyCaster(0.2).castTarget(owner, DUMMY_ENTANGLE_SPELL_ID, 1, Orders.entanglingroots, u, ownerUnit.getPos())
-                            snaredTargets.push(u)
+                            if new IttDummyCaster(0.2).castTarget(owner, DUMMY_ENTANGLE_SPELL_ID, 1, Orders.entanglingroots, u, ownerUnit.getPos())
```

Comment:
Shouldn't this not be using `ITTDummyCaster`?

Reply:
Updated